examples/quick_tutorial.ipynb

```diff
@@ -432,11 +434,233 @@
+   "outputs": [],
+   "source": [
+    "# fmt: off\n",
+    "PRIZES = [\n",
+    "    0,  # location 0 - a depot\n",
+    "    0,  # location 1 - a depot\n",
+    "    334,  # location 2\n",
+    "    413,  # location 3\n",
+    "    295,  # location 4\n",
+    "    471,  # location 5\n",
+    "    399,  # location 6\n",
+    "    484,  # location 7\n",
+    "    369,  # location 8\n",
+    "    410,  # location 9\n",
+    "    471,  # location 10\n",
+    "    382,  # location 11\n",
+    "    347,  # location 12\n",
+    "    380,  # location 13\n",
+    "    409,  # location 14\n",
+    "    302,  # location 15\n",
+    "    411,  # location 16\n",
+    "]\n",
+    "# fmt: on"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "When modelling optional clients, it is important to provide both a reward (the `prize` argument to `add_client`), and to mark the client as optional by passing `required=False`:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "m = Model()\n",
+    "depots = [\n",
+    "    m.add_depot(\n",
+    "        x=COORDS[idx][0],\n",
+    "        y=COORDS[idx][1],\n",
+    "        tw_early=TIME_WINDOWS[idx][0],\n",
+    "        tw_late=TIME_WINDOWS[idx][1],\n",
+    "    )\n",
+    "    for idx in range(2)\n",
+    "]\n",
+    "\n",
+    "for depot in depots:\n",
+    "    m.add_vehicle_type(2, depot=depot)\n",
+    "\n",
+    "clients = [\n",
+    "    m.add_client(\n",
+    "        x=COORDS[idx][0],\n",
+    "        y=COORDS[idx][1],\n",
+    "        tw_early=TIME_WINDOWS[idx][0],\n",
+    "        tw_late=TIME_WINDOWS[idx][1],\n",
+    "        prize=PRIZES[idx],\n",
+    "        required=False,\n",
+    "    )\n",
+    "    for idx in range(2, len(COORDS))\n",
+    "]\n",
+    "\n",
+    "locations = depots + clients\n",
+    "for frm_idx, frm in enumerate(locations):\n",
+    "    for to_idx, to in enumerate(locations):\n",
+    "        distance = abs(frm.x - to.x) + abs(frm.y - to.y)  # Manhattan\n",
+    "        duration = DURATION_MATRIX[frm_idx][to_idx]\n",
+    "        m.add_edge(frm, to, distance=distance, duration=duration)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "res = m.solve(stop=MaxRuntime(1), display=False)  # one second\n",
+    "print(res)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "_, ax = plt.subplots(figsize=(8, 8))\n",
+    "plot_solution(res.best, m.data(), plot_clients=True, ax=ax)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Some clients are not visited in the figure above.\n",
+    "These clients are all too far from the depots and other clients for their prizes to be worth more than the additional travel distance the visit would incur.\n",
```

Comment:
```suggestion
    "These clients are all too far from other locations to visit: their prizes are not worth the additional travel distance.\n",
```

Two suggestions: 
- the depots and other clients -> other locations
- (optional) some rephrasing to make the sentence shorter.